ontology/spdx-ontology.owl.xml

```diff
@@ -786,7 +824,9 @@ Known issues:</rdfs:comment>
         <rdfs:domain rdf:resource="http://spdx.org/rdf/terms#SpdxItem"/>
         <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#Literal"/>
         <rdfs:range rdf:resource="http://www.w3.org/2001/XMLSchema#string"/>
-        <rdfs:comment xml:lang="en">The text of copyright declarations recited in the Package or File.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The text of copyright declarations recited in the Package, File or snippet.
```

Comment:
For consistency, use `package, file` instead of `Package, File`. There are also more instances of incorrect capitalization.

Reply:
These should just be copy/paste from the spec.  I'll fix this one, but I probably won't have time to review the entire set of comments for capitalization.